tests/integration/ext4.rs

```diff
@@ -9,8 +9,17 @@
 use ext4_view::{Corrupt, Ext4, Ext4Error, Incompatible, Path, PathBuf};
 
 fn load_test_disk1() -> Ext4 {
-    const DATA: &[u8] = include_bytes!("../../test_data/test_disk1.bin");
-    Ext4::load(Box::new(DATA.to_vec())).unwrap()
+    let output = std::process::Command::new("zstd")
+        .args([
+            "--decompress",
+            // Write to stdout and don't delete the input file.
+            "--stdout",
+            "test_data/test_disk1.bin.zst",
+        ])
+        .output()
+        .unwrap();
+    assert!(output.status.success());
+    Ext4::load(Box::new(output.stdout)).unwrap()
```

Comment:
It's not obvious to me why this can't be deduplicated with the version in lib.rs. A comment pointing out the duplication might also be nice.

Reply:
Added duplication comments to both locations. In `tests/integration/ext4.rs`, explained the 'why': "We can't import that function here because it is gated by `cfg(test)`, and integration tests compile the crate-under-test without that config."